front-end/package.json

```diff
@@ -24,15 +33,62 @@
     "react-player": "^2.15.1",
     "react-resize-detector": "^10.0.1",
     "react-router-dom": "^6.22.3",
+    "tree-kill": "^1.2.2",
     "vite": "^5.4.3",
     "vite-plugin-commonjs": "^0.10.1",
     "vitest": "^2.1.1"
   },
   "scripts": {
     "start": "vite",
-    "build": "tsc && vite build",
+    "build:frontend": "tsc && vite build",
+    "build:electron": "electron-builder",
     "serve": "vite preview",
-    "lint": "npx eslint . --fix"
+    "lint": "npx eslint . --fix",
+    "electron": "electron src/electron/main.js --trace-warnings",
+    "dev": "concurrently \"npm run start\" \"npm run electron\""
```

Comment:
Dev is a pretty misleading name here if this is for electron dev stuff

Reply:
removed as it is not currently used